### ISSUE TITLE
Separate Initials repo

### ISSUE THREAD

**Issue by georgd** [georgd/EB-Garamond#168]
@timgrei and @Alexander-Wilms 

You might have noticed that I’ve created a new branch called “v2” with the first steps to unify Octavio’s and my versions. Going forward I think it’s a good idea to separate out the initials fonts (and ornaments?) to a new git repository. Thus we can treat them independently from the text fonts. For tools like the makefile this would simplify things, I think. What do you think?

**Comment by Alexander-Wilms** [comment]
It's fine by me.

I have no experience with building fonts from their source files, would the split simplify things due to the initials and ornaments being SVFs? In the master branch's Makefile I can't see a special case being made for the intials.

**Comment by georgd** [comment]
> I have no experience with building fonts from their source files, would the split simplify things due to the initials and ornaments being SVGs? In the master branch's Makefile I can't see a special case being made for the intials.

Yes and no. It would simplify the things because special treatment of the initials will be easier in general. But the SVGs have already been imported to the Fontforge sources so this doesn’t play a role.

**Comment by timgrei** [comment]
@Alexander-Wilms Could the fonts in other repos still be included in the release of this one?

**Comment by timgrei** [comment]
Then I think it's a good idea.

**Comment by georgd** [comment]
Done. Ypu find it under https://github.com/georgd/EB-Garamond-Initials

I’m transferring the issues right now.
